src/util/StringUtils.h

```diff
@@ -81,10 +81,10 @@ std::string lazyStrJoin(Range&& r, std::string_view separator);
 @brief Adds indentation before the given string and directly after new line
 characters.
 
-@param indentationLevel How deep is the indentation? `0` is no indentation.
+@param indentationSymbol What the indentation should look like..
 */
 inline std::string addIndentation(std::string_view str,
-                                  size_t indentationLevel);
+                                  std::string_view indentationSymbol = "    ");
```

Comment:
I don't think that there is a reasonable default for the indentation, so you should force the users to pass it in (4 spaces is rather random, but so is anything else that you could put here).

Reply:
Done, see [05f4941](https://github.com/ad-freiburg/qlever/pull/1010/commits/05f494113b2859836188b47d418854f38621ba5d).